deploy/testing-integration.yml

```
@@ -156,7 +177,7 @@ parameters:
 - name: IQE_TEST_IMPORTANCE
   value: 'critical'
 - name: IQE_SEL_IMAGE
-  value: 'quay.io/redhatqe/selenium-standalone:latest'
+  value: 'selenium/standalone-chrome:latest'
```

Please use: https://github.com/RedHatInsights/sre-testing-templates/blob/main/common/iqe-job-template.yaml#L154

```suggestion
  value: 'quay.io/cloudservices/selenium-standalone-chrome:136.0-20250828'
```